openmc/stats/multivariate.py

```diff
@@ -613,6 +616,117 @@ def from_xml_element(cls, elem):
         origin = [float(x) for x in elem.get('origin').split()]
         return cls(r, phi, z, origin=origin)
 
+class MeshIndependent(Spatial):
+    """Spatial distribution for a mesh.
+
+    This distribution allows one to specify a mesh to sample over and the scheme to sample over the entire mesh. 
```

Comment:
We should update this a bit now that we're just relying on `strengths` and `volume_normalized`.